src/python/pants_release/generate_github_workflows.py

```diff
@@ -1055,17 +1055,46 @@ def release_jobs_and_inputs() -> tuple[Jobs, dict[str, Any]]:
                         """
                     ),
                 },
+                {
+                    "name": "Make GitHub Release",
+                    "if": f"{IS_PANTS_OWNER} && steps.get_info.outputs.is-release == 'true'",
+                    "run": dedent(
+                        """\
+                        RELEASE_TAG=${{ steps.get_info.outputs.build-ref }}
+                        RELEASE_VERSION="${RELEASE_TAG#release_}"
+
+                        # NB: This could be a re-run of a release, in the event a job/step failed.
+                        if gh release view $RELEASE_TAG ; then
+                            exit 0
+                        fi
+
+                        GH_RELEASE_ARGS=("--notes" "")
+                        GH_RELEASE_ARGS+=("--title" "$RELEASE_TAG")
+                        if [[ $RELEASE_VERSION =~ [[:alpha:]] ]]; then
+                            GH_RELEASE_ARGS+=("--prerelease")
+                        else
+                            STABLE_RELEASE_TAGS=$(gh api -X GET -F per_page=100 /repos/{owner}/{repo}/releases --jq '.[].tag_name | sub("^release_"; "")')
+                            LATEST_TAG=$(echo "$STABLE_RELEASE_TAGS $RELEASE_TAG" | tr ' ' '\\n' | sort --version-sort | tail -n 1)
+                            if [[ $RELEASE_TAG == $LATEST_TAG ]]; then
+                                GH_RELEASE_ARGS+=("--latest")
```

Comment:
Per your comment in the PR description, it looks like this maps to [the `make_latest` parameter](https://github.com/cli/cli/blob/bf7db84ca8b795a38ee47b5e54a8109a917a55bf/pkg/cmd/release/create/create.go#L403-L404) in the REST API, and [its docs](https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#create-a-release) suggest that this has three values: true, false and legacy.

The Go code looks like it never sets "legacy", and the default is "true", but that's the one that deduces things based on date and version.

Thus, I wonder if the GH CLI docs are out of date.

Reply:
Oh I guess I never tried --latest, I just assumed it did what it says on the tin.

Let me try it locally to be sure